charts/nutanix-csi-storage/templates/ntnx-sc.yaml

```diff
@@ -59,8 +66,11 @@ parameters:
     nfsPath: {{ .Values.filePath }}
 reclaimPolicy: {{ .Values.fileClassRetention }}
 {{- end }}
----
+
 {{- if eq .Values.dynamicFileClass true }}
+---
```

Comment:
Again, why this change ( shifting --- below if statement) ?

Reply:
it was creating
---
---
in rendered file depending of SC choice